video/out/vo_dmabuf_wayland.c

```diff
@@ -539,6 +539,13 @@ static void resize(struct vo *vo)
                                                   lround(vo->dheight / wl->scaling_factor));
     wl_subsurface_set_position(wl->osd_subsurface, lround((0 - dst.x0) / wl->scaling_factor), lround((0 - dst.y0) / wl->scaling_factor));
     set_viewport_source(vo, src);
+
+    mp_mutex_lock(&vo->params_mutex);
+    vo->target_params = vo->params;
```

Comment:
It is only a pointer in vo.h. Need to have backing memory here in dmabuf_wayland, else we are editing params.

Reply:
Oh you're right. This is unexpectedly annoying. I'll have to keep a copy of the params in memory just for this.

Reply:
just put it inside `struct priv` and forget :)

Reply:
Hopefully less dumb now.